com/win32comext/shell/src/PyIQueryAssociations.cpp

```diff
@@ -45,8 +45,9 @@ PyObject *PyIQueryAssociations::Init(PyObject *self, PyObject *args)
     HRESULT hr;
     PY_INTERFACE_PRECALL;
     hr = pIQA->Init(flags, pszAssoc, hkProgid, hwnd);
-    PyWinObject_FreeWCHAR(pszAssoc);
     PY_INTERFACE_POSTCALL;
+    PyWinObject_FreeWCHAR(pszAssoc);
+    
```

Comment:
> And I don't know why clang-format has a problem.

You have some trailing whitespaces. You can run either of these commands to fix it locally:
- `clang-format -i $(git ls-files '*.cpp' ':!:com/win32comext/mapi/src/MAPIStubLibrary/')`
- `clang-format -i com/win32comext/shell/src/PyIQueryAssociations.cpp`
- `pre-commit run clang-format -a`

```suggestion

```

Reply:
Thank you for the hint. I ran pre-commit. And added the fix to `PyObject_CleanupDEFCONTEXTMENU` as well.